echopype/convert/api.py

```diff
@@ -334,6 +340,8 @@ def open_raw(
     xml_path: Optional["PathHint"] = None,
     convert_params: Optional[Dict[str, str]] = None,
     storage_options: Optional[Dict[str, str]] = None,
+    offload_to_zarr: Optional[bool] = False,
```

Comment:
```suggestion
    offload_to_zarr: bool = False,
```
`Optional` type hint means it can be `bool` or `None`.

Reply:
Ah, thank you for clarifying that. I misunderstood the Optional typing.